apps/portal/components/pagesComponents/_imagesScreen/pages/myImages/screenshotsContainer/ShareItemModal.tsx

```
@@ -63,13 +65,15 @@ const ShareItemModal: React.FC<IShareItemModalProps> = ({
-            Copy Link
+            {t('modals.copyLink')}
           </AppButton>
         </div>
       }
     >
-      <h2 className="tw-mb-3 tw-text-2xl tw-font-semibold">Share link</h2>
-      <label>Link to item</label>
+      <h2 className="tw-mb-3 tw-text-2xl tw-font-semibold">
+        {t('modals.shareLink')}
+      </h2>
+      <label>{t('modals.linkToItem')}</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)